cli/package.json

```diff
@@ -6,7 +6,6 @@
     "datastore-todos": "cli.js"
   },
   "dependencies": {
-    "gcloud-node-todos": "*",
```

Comment:
You should be able to keep it by adding the CLI to `.dockerignore` or moving the webapp to a sub dir.